core/src/table/table.stories.tsx

```diff
@@ -70,6 +70,51 @@ outer border or shadow. For that, use the \`Pane\` component.
 [2]: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
 `);
 
+
+export const Fill = () => {
+	// The definition of interface here is only for explanation purpose. In
+	// practice the interface/type/model should already be defined outside of
+	// your component.
+	interface Book {
+		isbn: number;
+		title: string;
+		author: string;
+	}
+
+	return (
+		<Table<Book>
+			fill
+			rows={[
+				{
+					isbn: 9780679783268,
+					title: "Pride and Prejudic",
+					author: "Jane Austen",
+				},
+				{
+					isbn: 9780743273565,
+					title: "The Great Gatsby",
+					author: "Francis Scott Fitzgerald",
+				},
+				{
+					isbn: 9780684830490,
+					title: "The Old Man and the Sea",
+					author: "Ernest Hemingway",
+				},
+			]}
+			rowKey={(book) => book.isbn.toString()}
+			columns={[
+				{ title: "Title", render: "title" },
+				{ title: "Author", render: "author" },
+			]}
+		/>
+	);
+};
```

Comment:
Should not copy the whole story like this. The point of this story is to demonstrate the "fill" prop, so the "rows" and "columns" and "interface" should come from outside constants. Only the "Basic" story should show everything. See the fixed and expandable stories